### DISCUSSION TITLE
Accept comments in `vercel.json`

### DISCUSSION THREAD

**Discussion by frencojobs** [vercel/community#1485] [Idea]
They can be really useful. Turborepo uses https://github.com/muhammadmuzzammil1998/jsonc (a JSONC parser) to deserialize from `turbo.json` and I've been leaving comments on pipelines and it's been very helpful. I tried to do the same with `vercel.json` today and realized it is not possible.

**Comment by slavafomin** [comment]
The comments in `vercel.json` are extremely needed, especially when you want to document the caching rules, that could get pretty hard to read. Right now, when I add an extra `"$comment": "Something"` property, Vercel fails to deploy the application.

**Reply by AlbertMarashi** [reply]
+1 here. At least allowing `$comment`  would be helpful, or supporting `.jsonc` for:
https://code.visualstudio.com/docs/languages/json#_json-with-comments

**Comment by johnitvn** [comment]
any update here?
